orchestrator/orchestrator.go

```diff
@@ -119,63 +150,77 @@ func (s *Orchestrator) execute(filter func(wf *process.Process, node *process.Pr
 	var processes []*process.Process
 	route := fmt.Sprintf("custom/%s/%s", processmodule.QuerierRoute, processmodule.QueryList)
 	if err := s.rpc.QueryJSON(route, nil, &processes); err != nil {
-		s.ErrC <- err
+		s.logger.Error(err.Error(), err)
```

Comment:
why this one is logged on the logger but not the pubsub. I would put everything on the pubsub

Reply:
the pubsub is reserved for error related to a process. In this case the error is not related to any pubsub, so it is only displayed.

Reply:
The goal of the pubsub + gRPC stream is to expose to users useful info concerning their process.
To get ALL the logs of the orchestrator, lcd, validator, rpc, users should check the logs of the docker (and I'm working on putting the log of liteflow staying and prod on cloud log so it's easy to access).